### ISSUE TITLE
Light Theme

### ISSUE THREAD

**Issue by S-Man42** [S-Man42/GCWizard#158]
Some (mainly older) users reported, it were hard for them to read the dark UI. 

For improved accessibility a light theme should be optional available.

**Comment by S-Man42** [comment]
#190
